README.md

```diff
@@ -42,7 +42,7 @@ service('settings')->forget('App.siteName');
 
 ## Docs
 
-Read the full documentation: https://codeigniter4.github.io/settings/
+Read the full documentation: https://settings.codeigniter.com
```

Comment:
If you are going to use https://settings.codeigniter.com, it is better to use it in all cases. including file composer.js & mkdocs.yml and...

Reply:
IMO composer.json should link to the source code only.
As for MkDocs... do you mean instead of linking to codeigniter.com in `extra.homepage`?

Reply:
> IMO composer.json should link to the source code only. 
Thanks, I didn't know that.

> As for MkDocs... do you mean instead of linking to codeigniter.com in `extra.homepage`?
Sorry.  No, it's fine now, I thought there was a link to the docs defined in MkDocs.